b2b-sample/guardio-insurance-sample-application/components/settingsComponents/settings.js

```diff
@@ -41,20 +41,19 @@ export default function Settings(props) {
 
     const [activeKeySideNav, setActiveKeySideNav] = useState('1');
 
-    const signOutOnClick = () => orgSignout();
-
     const mainPanelComponenet = (activeKey, session) => {
         switch (activeKey) {
             case '1':
+
                 return <HomeComponent orgName={props.name} orgId={props.orgId} session={session} />;
             case '2-1':
+
                 return <ViewUserComponent orgName={props.name} orgId={props.orgId} session={session} />;
-            case '2-2':
-                return <AddUserComponent orgName={props.name} orgId={props.orgId} session={session} />;
             case '2-3':
+                
                 return <IdentityProviders orgName={props.name} orgId={props.orgId} session={session} />;
-            case '3-1':
-                return <Application orgName={props.name} session={session} />
+            // case '3-1':
+            //     return <Application orgName={props.name} session={session} />
```

Comment:
Shall we remove this if we don't need other than commenting?

Reply:
This file has been changed with new PR's and these errors has been fixed,

### Commit
- https://github.com/Achintha444/samples-is/commit/3a945da41d8b968ff09ee7268090cc91dd786eb7#diff-de6e61ef1f056a6fd471d69dbf61ddf99deb9ca43519917d6e2dc1f5283551b4